core/src/main/java/hudson/cli/declarative/CLIRegisterer.java

```diff
@@ -195,10 +201,33 @@ public int main(List<String> args, Locale locale, InputStream stdin, PrintStream
                                     sc.setAuthentication(old); // restore
                                 }
                             } catch (CmdLineException e) {
-                                stderr.println(e.getMessage());
-                                printUsage(stderr,parser);
-                                return 1;
+                                stderr.println("\nERROR: " + e.getMessage());
+                                printUsage(stderr, parser);
+                                return 2;
+                            } catch (IllegalStateException e) {
+                                stderr.println("\nERROR: " + e.getMessage());
```

Comment:
`\n` is platform specific. Should use `System.getProperty("line.separator")` (or `%n` in `printf`).